examples/SConstruct

```diff
@@ -92,7 +92,7 @@ prgs = []
 for e, extralibs in examples:
     prgs.append(env.Program(e, LIBS=[
         parameters,
-        'boost_thread-mt',
```

Comment:
In Linux, 'mt' suffix is not generated in build process (it's just 'boost_thread'), because '-mt' suffix has been removed from Boost (already multi-threading safe).